src/components/contact/RedditCard.tsx

```diff
@@ -12,12 +12,12 @@ const RedditCard = () => (
       <Reddit />
     </div>
     <div className='card__body'>
-      Participate in community discussion and troubleshooting and receive updates from the Jellyfin project and
-      community on Reddit.
+      r/Jellyfin has <a href="https://www.theverge.com/2023/6/5/23749188/reddit-subreddit-private-protest-api-changes-apollo-charges">gone private</a>. Our chat options remain open.
+      We are currently on Lemmy: <a href="https://beehaw.org/c/jellyfin@lemmy.ml">c/Jellyfin</a> (via Beehaw)
     </div>
     <div className='card__footer'>
-      <a href='https://www.reddit.com/r/jellyfin' className='button button--block button--reddit'>
-        /r/jellyfin
+      <a href='#' className='button button--block button--reddit'>
```

Comment:
```suggestion
      <button className='button button--block button--reddit'>
```

The closing tag needs changed too but I can't add that as a suggestion. 😅